Marlin/src/lcd/extui/lib/mks_ui/tft_lvgl_configuration.cpp

```diff
@@ -123,6 +125,10 @@ void tft_lvgl_init() {
 
   watchdog_refresh();     // LVGL init takes time
 
+  #if MB(MKS_ROBIN_NANO)
+    OUT_WRITE(PB0, LOW);      // HE1
+  #endif
```

Comment:
@MKS-Sean — Is this pin prone to being set HIGH on boot? If so, what is the result of that? If this isn't directly associated with the `TFT_LVGL_UI` but more related to the board and its initialization, it may be defined instead as `BOARD_INIT` in the relevant board…
```cpp
#define BOARD_INIT() OUT_WRITE(PB0, LOW);
```
…and then it will be initialized early in the `setup()` sequence.